### PR TITLE
debug: Log X-Forwaded-For for requests (PROJQUAY-2883)

### PR DESCRIPTION
**Pull request by syed**
This for debugging issues with ALB in quay.io

### CODE REVIEW

**Review by bcaton85 — DISMISSED** [review]
LGTM

File: app.py [hunk 1]
```diff
@@ -175,9 +174,14 @@ def _request_start():
...
         request.request_id,
         request.path,
-        extra={"request_id": request.request_id},
+        extra={
+            "request_id": request.request_id,
+        },
     )
 
+    x_forwarded_for = request.headers.get("X-Forwarded-For", None)
+    logger.debug("[%s] X-Forwarded-For: %s" % (request.request_id, x_forwarded_for))
```
Comment (kleesc) [inline app.py:183]: Can you add this as part of the above debug statement?
```   
debug_extra = {
    "request_id": request.request_id,
}
if x_forwarded_for is not None:
    debug_extra["X-Forwarded-For"] = x_forwarded_for

logger.debug(
    "Starting request: %s (%s)",
    request.request_id,
    request.path,
    extra=debug_extra,
)
```

**Review by kleesc — CHANGES_REQUESTED** [review]

File: app.py [hunk 2]
```diff
@@ -123,7 +122,7 @@
 # If the "preferred" scheme is https, then http is not allowed. Therefore, ensure we have a secure
 # session cookie.
 if app.config["PREFERRED_URL_SCHEME"] == "https" and not app.config.get(
-    "FORCE_NONSECURE_SESSION_COOKIE", False
+        "FORCE_NONSECURE_SESSION_COOKIE", False
```
Comment (kleesc) [inline app.py:125]: Oops. Formatting space/tabs probably

Comment (kleesc) [reply app.py:125]: Otherwise LGTM

Comment (kleesc) [reply app.py:125]: You can just run `make black` to format everything.

**Review by kleesc — APPROVED** [review]
LGTM

**Pull request merged**